dashboard/app/models/script_level.rb

```diff
@@ -136,12 +136,8 @@ def end_of_script?
   end
 
   def long_assessment?
-    if assessment
-      if level.properties["pages"] && level.properties["pages"].length > 1
-        return true
-      end
-    end
-    false
+    return false unless assessment
```

Comment:
Actually, on further thought, this changes the method definition. If level.properties["pages"] is nil, for example, the method returns nil rather than false. Right?